docs/source/project_documentation/dev_guide/validation_plugins.md

```diff
@@ -1,21 +1,12 @@
 # Validation Plugins for Developers
 
-## General Information surrounding Validation
+## Plugin-provided Validations
+AaC's approach to validation leverages the larger AaC plugin mechanism to provide a plug-and-play system for creating structural and primitive type validations (constraints) that are used to detect errors and possible issues in systems modeled with AaC.
 
-Any references to code within this document regarding the `validation` plugin can be found here in the AaC project:
-`AaC/python/src/validate/_validate.py`
-
-This validator plugin can be separated into the below sections.
-
-1. [Definition Validations](#definition-validations)
-2. [Primitive Validations](#primitive-validations)
+AaC makes use of two classes of plugin-provided validations: definition validations and primitive validations. The [definition validations](#definition-validations) are used to validate the structural parts of definitions such as required fields while [primitive validations](#primitive-validations) check that values in primitive fields adhere to the expectations of the data type such as checking that values listed as integers are, in fact, integers.
 
 There is also a section listing out the `validation` plugin for the implementation of a validator plugin for 1st and/or 3rd party plugins
```

Comment:
I think the linkage to the lower section should get to remain, think it got removed accidentally in the old PR that we pulled from. 

```suggestion
There is also a section listing out the `validation` plugin for [implementation of a validation plugin for definitions](#implementation-of-a-validation-plugin-for-definitions) for 1st and/or 3rd party plugins.
```